src/applications/vre/components/GetFormHelp.jsx

```diff
@@ -1,25 +1,17 @@
 import React from 'react';
 import Telephone, {
   CONTACTS,
+  PATTERNS,
 } from '@department-of-veterans-affairs/formation-react/Telephone';
 
-export default function GetFormHelp() {
-  return (
-    <div>
-      <p className="help-talk">Enrollment or Eligibility questions:</p>
-      <p className="help-phone-number">
-        <a className="help-phone-number-link" href="tel:1-877-222-8387">
-          877-222-8387
-        </a>
-        <br />
-        TTY:{' '}
-        <Telephone
-          className="help-phone-number-link"
-          contact={CONTACTS.HELP_TTY}
-        />
-        <br />
-        Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. ET
```

Comment:
Don't need the day & times anymore?

Reply:
Following these mockups - https://preview.uxpin.com/e291c4ac8956d804d774160cdb82ecb724044689#/pages/130773489/simulate/sitemap